package.json

```diff
@@ -11,10 +11,13 @@
     "url": "https://github.com/Shoptet/microsite-styles/issues"
   },
   "devDependencies": {
-    "grunt": "^1.0.2",
-    "grunt-cli": "^1.2.0",
-    "grunt-contrib-sass": "^1.0.0",
-    "grunt-contrib-watch": "^1.0.1",
-    "load-grunt-tasks": "^4.0.0"
+    "grunt": "^1.4.1",
+    "grunt-cli": "^1.4.3",
+    "grunt-contrib-watch": "^1.1.0",
+    "load-grunt-tasks": "^5.1.0",
+    "sass": "^1.38.2"
+  },
+  "dependencies": {
+    "grunt-dart-sass": "^2.0.1"
```

Comment:
This should be in` devDependencies` too.

Reply:
Done, thank you.